docs/how-to-guides/running-feast-in-production.md

```diff
@@ -51,9 +53,29 @@ Notice how the registry has been configured to use a Google Cloud Storage bucket
 It is important to note that the CI system above must have access to create, modify, or remove infrastructure in your production environment. This is unlike clients of the feature store, who will only have read access.
 {% endhint %}
 
+If your organization consists of many independent Data Science teams or single team is working on several projects
+that could benefit from sharing features / entities / sources / transformations we encourage you to utilize Python packages inside each environment:
```

Comment:
any thoughts on how collaboration would work here though?

e.g. if we have multiple models in production relying on materialized feature views, we probably shouldn't let users really modify feature views since that can cause issues for those model services right?

Reply:
```suggestion
that could benefit from sharing features, entities, sources, and transformations, then we encourage you to utilize Python packages inside each environment:
```